frontend/talentsearch/resources/js/components/workPreferencesForm/WorkPreferencesForm.tsx

```diff
@@ -0,0 +1,156 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import React from "react";
+import { useIntl } from "react-intl";
+import { toast } from "react-toastify";
+import { errorMessages } from "@common/messages";
+import { BasicForm, Checklist, RadioGroup } from "@common/components/form";
+import { getLocale } from "@common/helpers/localize";
+import { notEmpty } from "@common/helpers/util";
+import {
+  OperationalRequirement,
+  useGetOperationalRequirementsQuery,
+} from "../../api/generated";
+import ProfileFormWrapper from "../applicantProfile/ProfileFormWrapper";
+import ProfileFormFooter from "../applicantProfile/ProfileFormFooter";
+
+export const WorkPreferencesForm: React.FunctionComponent<{
+  operationalRequirements: OperationalRequirement[];
+  handleSubmit: (data: any) => Promise<void>;
```

Comment:
I will try, may need help on this.

Reply:
@petertgiles 
How can ditermine the formvalues type?
Is there is any input type in graphql schema or I need to write it? 
I couldn't find any corresponding database table for work preferences.
For handleSubmit Do I need to write mutation for handleSubmit I need help for that. Formvalues should store in database tabel?

Reply:
Yep, you will write the type since it is specific to just this form.  The one for CreateRequest is here: https://github.com/GCTC-NTGC/gc-digital-talent/blob/1a9092d6611c1d27978feea9c17eb234a1d63f7b/frontend/talentsearch/resources/js/components/request/CreateRequest.tsx#L32